compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -1609,6 +1619,13 @@ pub struct Coverage {
     pub code_region: CodeRegion,
 }
 
+#[derive(Clone, Debug, PartialEq, TyEncodable, TyDecodable, HashStable, TypeFoldable)]
+pub struct CopyNonOverlapping<'tcx> {
+    pub src: Place<'tcx>,
+    pub dst: Place<'tcx>,
```

Comment:
Both of these should be operands (i.e. much the same thing as function arguments) not places (things that appear on the left side of the assignment operator).